spec/project_spec.rb

```diff
@@ -676,6 +676,21 @@ class InvalidErrorHandler; end
       expect(spy_logger).to have_received(:log).once.with(Logger::INFO, "Feature 'boolean_single_variable_feature' is enabled for user 'test_user'.")
     end
 
+    it 'should return true if the user is bucketed into a feature rollout given featureEnabled property is false' do
+      experiment_to_return = config_body['rollouts'][0]['experiments'][0]
+      variation_to_return = experiment_to_return['variations'][0]
+      decision_to_return = Optimizely::DecisionService::Decision.new(
+        experiment_to_return,
+        variation_to_return,
+        Optimizely::DecisionService::DECISION_SOURCE_ROLLOUT
+      )
+      allow(project_instance.decision_service).to receive(:get_variation_for_feature).and_return(decision_to_return)
```

Comment:
Set variation's 'featureEnabled' property to false here. Or assert that it's false to prove your claim

Reply:
Done